guides/common/modules/proc_installing-webhooks-shellhooks-plugin.adoc

```diff
@@ -0,0 +1,59 @@
+[id="installing-webhooks-shellhooks-plugin_{context}"]
+= Installing Webhooks and Shellhooks Plugin
+
+.Procedure
+
+ifdef::satellite[]
+
+* Install the webhooks plugin using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-installer --enable-foreman-plugin-webhooks 
+----
+
+* Install the shellhooks plugin on each {SmartProxy} that will be used for shellhooks, using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-installer --enable-foreman-proxy-plugin-shellhooks
+----
+
+* Install the CLI plugin using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum install tfm-rubygem-hammer_cli_foreman_webhooks
+----
+
+endif::[]
+
+
+ifdef::foreman-el,foreman-deb,katello[]
+
+When using Foreman 2.4 or Katello 4.0, installer is not currently available for the webhooks and shellhooks plugin. For a manual installation of the webhooks and shellhooks plugin, run the following commands:
+
+. Install the rubygem packages. The `foreman-maintain` tool will install packages, execute `foreman-installer` to apply all configuration changes, migrate and seed the database.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-maintain packages install tfm-rubygem-foreman_webhooks \
+  tfm-rubygem-smart_proxy_shellhooks
+----
+. Optional: Enable the shellhooks plugin.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# grep enabled /etc/foreman-proxy/settings.d/shellhooks.yml
+:enabled: https
+----
+. Restart the {SmartProxy}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl restart foreman-proxy
+----
+
+In the {Project} web UI, navigate to *Infrastructure* > *Smart Proxies* > *Refresh features* to allow shellhooks feature to show up.
+
+endif::[]
```

Comment:
This foreman part is no longer valid, Foreman 2.4 is long gone. We can just use the installer part and remove "if satellite" from there.